handler/http/client/client.go

```diff
@@ -1,25 +1,52 @@
-package handler
+package client
 
 import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"net/http"
 	"strings"
 
+	"github.com/DATA-DOG/godog"
 	"github.com/DATA-DOG/godog/gherkin"
 	"github.com/olekukonko/tablewriter"
 	"github.com/tomatool/tomato/compare"
 	"github.com/tomatool/tomato/errors"
+	"github.com/tomatool/tomato/resource"
 )
 
+type HTTPClient interface {
```

Comment:
I don't think this should be exported, probably better to have HTTPClient implemented wherever this is used, not in the producer

Reply:
Never mind, I'm wrong :)

Reply:
This should probably be just Client as well